CONTRIBUTING.md

```diff
@@ -1,20 +1,49 @@
 How to contrubute to SunPy
 --------------------------
 
+Hello, and thanks for being interested in contributing to SunPy!
+
 SunPy is an open project that enourages everyone to contribute in any way possible.
-This document descibes how to contribute code and documentation to the main SunPy 
-repository, if you wish to make a different contribution please contact the 
-[mailing list](https://groups.google.com/forum/#!forum/sunpy).
+This document mainly focuses on code or documentation contributions, however
+there are many other ways to get involved with the SunPy community, a good place
+to start is saying hello on the [mailinglist]() or [IRC]().
+
+
+## Reporting Issues
+
+If you have found a problem when using SunPy the best way to report it and get
+some help is by opening an [issue](https://github.com/sunpy/sunpy/issues) on our
+GitHub issue tracker.
+
+A good issue will provide instructions and a code sample on how someone else can
```

Comment:
Issue reports should provide a short description of the issue.  If you are reporting a bug, please post a code sample so others can attempt to reproduce the error.  Please also post any error output generated by the bug.  For a good example of how to do this see +issue [#1191](https://github.com/sunpy/sunpy/issues/1191).

If you are making a feature request, please post as much information as possible regarding the feature you would like to see in SunPy.